third_party/move/move-bytecode-verifier/src/verifier.rs

```diff
@@ -265,7 +272,7 @@ impl VerifierConfig {
             max_basic_blocks: Some(1024),
             max_basic_blocks_in_script: Some(1024),
             max_value_stack_size: 1024,
-            max_type_nodes: Some(256),
+            max_type_nodes: Some(128),
```

Comment:
@georgemitenkov I don't think this needs feature gating.

Reply:
Why? What if someone publishes code with 200 nodes during rollout?

Reply:
Ah yes, did not see that this is a  `production()` method